testdata/kube-config.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2021 The Kubermatic Kubernetes Platform contributors.
```

Comment:
As stated in the previous comment please move this file to the path:pkg/controllers/osc/testdata/